lib/va_profile/veteran_status/service.rb

```diff
@@ -83,10 +82,6 @@ def handle_client_error(e)
         handle_error(e)
       end
 
-      def edipi_present!
-        raise 'User does not have a valid edipi' if @user&.edipi.blank?
-      end
-
```

Comment:
removing this method because we are now checking for a blank edipi before we get here - that will eliminate the raising of an error that reads as 296 status code after.